index.html

```diff
@@ -24,6 +24,60 @@
 </head>
 
 <body>
+
+    <div id="mobile-details-modal" class="mobile-modal hidden" >
+        <div class="mobile-details-container" >
+            <div class="mobile-details-content">
+                <img id="close-detail" src="./Enabled_1.png" alt="close-icon" />
+            </div>
+            <div class="heading-and-button-group">
+                <h2 class="detail-popup-heading">
+                    **heading**
+                </h2>
+                <div class="popup-button-group-desktop">
+                    <button type="button" class="btn-popup">
+                        See Live
+                        <img alt="shortcut" class="icon-btn" src="./exp.png" />
+                    </button>
+                    <button type="button" class="btn-popup">
+                        See Source
+                        <img alt="icon" class="icon-btn" src="gitexp.png" />
```

Comment:
- [ ] Please make sure that in mobile view the content inside the buttons does not wrap: 

<img width="338" alt="Screenshot 2022-03-10 at 13 48 32" src="https://user-images.githubusercontent.com/77038610/157635931-c5172803-ac86-4d69-8bb2-63c4e35e716d.png">
 You can achieve that by decreasing the font size, deleting the letter spacing, and decreasing the padding. 
Try this: 

```
.popup-button-group > .btn-popup {
    font-size: 15px;
    letter-spacing: 0;
    justify-content: center;
    align-items: center;
}
```